### ISSUE TITLE
haste-boot fails on Mac OS X 10.9

### ISSUE THREAD

**Issue by rcolinray** [valderman/haste-compiler#150]
I installed using `cabal update` followed by `cabal install haste-compiler`. However, `haste-boot` fails with the following message:

```
Warning: Instead of 'ghc-options: -XCPP' use 'extensions: CPP'
cabal: Missing dependency on a foreign library:
* Missing (or bad) header file: HsBase.h
This problem can usually be solved by installing the system package that
provides this library (you may need the "-dev" version). If the library is
already installed but in a non-standard location then you can use the flags
--extra-include-dirs= and --extra-lib-dirs= to specify where it is.
If the header file does exist, it may contain errors that are caught by the C
compiler at the preprocessing stage. In this case you can re-run configure
with the verbosity flag -v3 to see the error messages.
Command '/Users/colin/.cabal/bin/haste-inst' failed with error  code 1
```

Full gist: https://gist.github.com/rcolinray/9928335

**Comment by valderman** [comment]
Assuming this is fixed since I can't reproduce it and there's been no word that someone else can. Reopen with new repro instructions if this assumption is wrong.
